.github/workflows/build.yaml

```diff
@@ -40,10 +40,14 @@ jobs:
         uses: actions/checkout@v3
 
       - name: "Build with Bazel"
+        env:
+          CC: ${{ matrix.compiler }}
```

Comment:
I think I've had situation where I had to set both CC and CXX for Bazel to actually use the compilers I specified and not default to gcc, but not 100% certain about this.

Reply:
Thanks! Let's try to specify both.